### PR TITLE
Feature/improved fetcher

### PR DESCRIPTION
**Pull request by msusicky**
Better fetcher included, table in statistics enhanced with the column vymazani. News page updated.

### CODE REVIEW

File: app/templates/macros/tables/stav_registraci_kategorie.html [hunk 1]
```diff
@@ -14,7 +14,13 @@
             <th scope="col" class="w-20 text-center" colspan="1">
                 S termínem
                     <i class="fas fa-info-circle" data-toggle="tooltip" data-placement="auto" data-html="true"
-                       title="Počet lidí s PIN2, z nichž někteří již byli očkováni - nelze rozlišit">
+                       title="Počet lidí s PIN2, kteří ještě pravděpodobně nebyli očkováni - možná nepřesnost 1-2 dní">
+                    </i>
+            </th>
+            <th scope="col" class="w-20 text-center" colspan="1">
+                Vymazaní
```
Comment (jan-stanek) [inline app/templates/macros/tables/stav_registraci_kategorie.html:21]: Očkovaní?

File: app/templates/statistiky.html [hunk 2]
```diff
@@ -96,6 +96,7 @@ <h5>Nedovolené zjednodušení</h5>
                         title="Dataset registrovaných dle profesí.">Stav registrací dle prioritních skupin</h5>
                     {{ stav_registraci_kategorie(reservations_category) }}
                     <p class="card-text font-italic mt-3">
+                        28. 4. z historických dat jsme rekonstruovali staré údaje. (data ze systému mizí po očkování) <br />
```
Comment (jan-stanek) [inline app/templates/statistiky.html]: I would remove both notes and leave it only in news

File: app/fetcher/registrations_fetcher.py [hunk 3]
```diff
@@ -13,16 +15,42 @@ class RegistrationsFetcher(Fetcher):
...
-        df = df.drop(['ockovaci_misto_nazev', 'kraj_nuts_kod', 'kraj_nazev'], axis=1)
+        if 'OckovaciCentrumKod' in df:
+            # Tailor the dataset to the open one
+            df['Datum'] = df['Datum'].astype(str).str[:10]
+            df = df.rename(columns={'Datum': 'datum', 'OckovaciCentrumKod': 'ockovaci_misto_id'
+                , 'VekovaSkupina': 'vekova_skupina', 'PovolaniNazev': 'povolani'
+                , 'Zeme': 'stat', 'Rezervace': 'rezervace'
+                , 'DatumRezervace': 'datum_rezervace'})
+            # We import only notblocked or blocked with vaccination
+            df = df.loc[df['Zruseno'] == 'Ne'].loc[
```
Comment (jan-stanek) [inline app/fetcher/registrations_fetcher.py:35]: it should be more effective as one condition

File: app/templates/statistiky.html (hunk 2 above)
Comment (msusicky) [reply app/templates/statistiky.html]: Good idea

Comment (jan-stanek) [reply app/templates/statistiky.html]: The same note is probably also below some graph, remove it too please

**Review by jan-stanek — APPROVED** [review]

**Pull request merged**